app/server/app/public/data/national/NARS.json

```
@@ -38,31 +38,31 @@
+          "content": "<p>Microcystin is a cyanotoxin that is produced by naturally-occurring bacteria in surface waters. Sampling also shows that 2% of the nation’s lakes had levels that could pose risks to people swimming or playing in the water.</p><p>At high levels, cyanotoxins can present a risk to public drinking water systems and to people, pets, and livestock.</p><p><a href='https://www.epa.gov/system/files/documents/2022-08/When%20Good%20Algae%20Go%20Bad_2017.pdf' target='_blank' rel='noopener noreferrer'>More Information (PDF)</a> <small>(opens new browser tab)</small></p>"
         }
       ],
       "footnotes": [
         "This data is pulled from the National Aquatic Resource Surveys (NARS) and the metrics are only for the conterminous US."
       ]
     },
     {
-      "title": "Coasts",
+      "title": "Coastal",
       "content": [
         {
           "title": "<span class='hmw-percent'>71%</span> of our estuaries and <span class='hmw-percent'>31%*</span> of Great Lakes nearshore waters are healthy based on their biological communities.",
           "content": "<p>In estuaries, the percentage of area in good condition increased from 51% to 71% from 2005-06 to 2015.  Biological condition tells us how healthy a waterbody is. A healthy waterbody supports aquatic communities – such as worms, snails, and clams – that are sensitive to changes in their environment. Their presence or absence gives us an idea of how healthy or degraded our waters are.</p>"
         },
         {
-          "title": "<span class='hmw-percent'>17%</span> of our estuaries and <span class='hmw-percent'>23%</span> of Great Lakes nearshore waters were in poor condition due to excess nutrients.",
-          "content": "<p>While nutrients are important, having too many nutrients is bad. Excess nutrients come from farm fertilizer, wastewater treatment, atmospheric deposition, animal manure, and urban runoff, and cause problems for water quality. Excess nutrients can lead to algal blooms and fish kills, leading to a loss of fishing, recreational, and tourism opportunities.</p>"
+          "title": "<span class='hmw-percent'>17%</span> of our estuaries and <span class='hmw-percent'>23%</span> of Great Lakes nearshore waters have excess nutrients.",
+          "content": "<p>While nutrients are important, having too many nutrients can cause problems for water quality. Excess nutrients come from farm fertilizer, wastewater treatment, atmospheric deposition, animal manure, and urban runoff, and cause problems for water quality. Excess nutrients come from farm fertilizer, animal manure, wastewater treatment, atmospheric deposition, and urban runoff. Excess nutrients can lead to algal blooms and fish kills, leading to a loss of fishing, recreational and tourism opportunities</p>"
```

```suggestion
          "content": "<p>While nutrients are important, having too many nutrients can cause problems for water quality. Excess nutrients come from farm fertilizer, animal manure, wastewater treatment, atmospheric deposition, and urban runoff. Excess nutrients can lead to algal blooms and fish kills, leading to a loss of fishing, recreational and tourism opportunities</p>"
```

The same sentence was in here twice with slightly different ordering.

---

Thanks for fixing that.